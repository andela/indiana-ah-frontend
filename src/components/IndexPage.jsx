import React from 'react';

const IndexPage = () => (
  <div className='container'>
    <h2>Welcome to Author's Haven!</h2>
  </div>
);

export default IndexPage;

