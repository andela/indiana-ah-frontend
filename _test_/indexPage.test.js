import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { shallow, mount, render } from 'enzyme';
import renderer from 'react-test-renderer';
import IndexPage from '../src/components/IndexPage.jsx';

describe('Custom Footer component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<IndexPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders a footer', () => {
    const indexPage = shallow(<IndexPage />);
    expect(indexPage.find('.container')).toBeDefined();
    const h2 = indexPage.find('h2');
    expect(h2.length).toEqual(1);
    expect(h2.text()).toEqual('Welcome to Author\'s Haven!');
  });
});