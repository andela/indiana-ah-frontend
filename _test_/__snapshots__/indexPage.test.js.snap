// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Footer component renders correctly 1`] = `
<div
  className="container"
>
  <h2>
    Welcome to Author's Haven!
  </h2>
</div>
`;
